src/octokit/get-probot-octokit-with-defaults.ts

```diff
@@ -1,6 +1,7 @@
 import type { LRUCache } from "lru-cache";
 import { ProbotOctokit } from "./probot-octokit.js";
 import type { RedisOptions } from "ioredis";
+import { request } from "@octokit/request";
```

Comment:
This should be added as a dependency to `package.json`. Currently we rely on it being a dependency of `@octokit/core`, and not as a direct dependency of probot